app/controllers/resources_controller.rb

```diff
@@ -6,12 +6,29 @@ def index
                 else
                   Resource.all
                 end
-
-    render json: resources.to_json(include: ['addresses', 'phones', 'categories', { schedule: { include: 'schedule_days' } }])
+    render json: resources.to_json(RENDERED_RESOURCE_ATTRIBUTES)
   end
 
   def show
     resource = Resource.find(params[:id])
-    render json: resource.to_json(include: ['addresses', 'phones', 'categories', { schedule: { include: 'schedule_days' } }])
+    render json: resource.to_json(RENDERED_RESOURCE_ATTRIBUTES)
   end
+
+  # rubocop:disable MutableConstant
```

Comment:
so rubocop wants me to add the .freeze because it thinks that these should be immutable, but its causing

Completed 500 Internal Server Error in 147ms (ActiveRecord: 11.4ms)

RuntimeError (can't modify frozen Hash):
app/controllers/resources_controller.rb:14:in `show'

When invoked, Ill have to have it ignore the freeze/mutability check.